pkg/config/systemd.go

```diff
@@ -56,3 +61,19 @@ func useSystemd() bool {
 	})
 	return usesSystemd
 }
+
+func useJournald() bool {
+	journaldOnce.Do(func() {
+		if !useSystemd() {
+			return
+		}
+		journal, err := sdjournal.NewJournal()
+		if err != nil {
```

Comment:
@giuseppe @vrothberg @nalind Do you know if this is enough to check if we can use the journal?

Reply:
It looks like the implementation will either manage to open the journal for reading and return a handle for it, or return an error.  The APIs for sending data to the journal don't go through the handle, so I think we have to take it on faith that messages we send to the journal will be retrievable through the handle.

Reply:
SGTM